dockstore-client/src/main/java/io/dockstore/client/cli/nested/WESLauncher.java

```diff
@@ -125,4 +128,18 @@ public void runWESCommand(String jsonInputFilePath, File localPrimaryDescriptorF
             }
         }
     }
+
+    /**
+     * help text output
+     */
+    private void wesCommandSuggestions(String runId) {
+        String wesEntryType = abstractEntryClient.getEntryType().toString().toLowerCase();
+        out("Useful commands for working with WES runs:");
+        out("");
+        out("To get status for this run: dockstore " + wesEntryType + " wes status --id " + runId
```

Comment:
Is the value of `wesEntryType` something like CWL or WDL? Peeked briefly at swagger, and I don't see such a parameter for this endpoint.

If I'm wrong, ignore. If I'm right:

1. Remove it from message
2. Remove from command/help

Reply:
wesEntryType is either 'workflow' or 'tool'. We don't support tool, and the option to use 'tool' is short circuited earlier in the control flow, so it will never be 'tool' unless someday we do support 'tool'

Reply:
Ah, my bad... That said, it still seems sort of unnecessary, unless there will ever be a day where:

1. We support tools, AND
2. Specifying tool vs workflow as a command-line argument will return a different result for the same run id.

On the other hand, if this breaks the symmetry of how commands work, then maybe we need it, but it feels odd to force the user to supply an argument that that will never have any impact (unless I'm wrong in my second point).

Reply:
Yes it fits will how our command line is structured to launch tools or workflows. I suppose  we could just hardcode 'workflow', but I will just leave it for now.

Reply:
To be clear, I was not arguing to drop `workflow` everywhere. It makes sense to have it for launch, even if we don't yet support tools. But it adds no value and just gets in the way on the status or cancel commands, which only require the id.

I guess we can circle back to if we take this feature out of beta.